src/app/_components/DesktopNavigation.tsx

```diff
@@ -0,0 +1,181 @@
+'use client'
+
+import Link from 'next/link'
+import { usePathname } from 'next/navigation'
+
+import { ArrowUpRight } from '@phosphor-icons/react'
+import clsx from 'clsx'
+import { Route } from 'next'
+
+import { Icon } from '@/components/Icon'
+import { NavigationPopover } from '@/components/NavigationPopover'
+
+import { PATHS } from '@/constants/paths'
+
+const getInvolvedItems = [
+  {
+    label: PATHS.EVENTS.label,
+    href: PATHS.EVENTS.path,
+    description: 'Overview of upcoming and past events',
+  },
+  {
+    label: PATHS.GRANTS.label,
+    href: PATHS.GRANTS.path,
+    description:
+      'Information on funding opportunities supporting projects that contribute to the growth of the Filecoin Network',
+  },
+  {
+    label: 'Become a Storage Provider',
+    href: 'https://destor.com/destor-network/overview',
+    description: 'Dive right in and become an essential part of the ecosystem',
+  },
+  {
+    label: 'Contribute to the Filecoin Project',
+    href: 'https://docs.filecoin.io/basics/project-and-community/ways-to-contribute',
+    description:
+      'Shape the future of Filecoin by contributing to its code, research, or docs.',
+  },
+]
+
+const communityItems = [
+  {
+    label: PATHS.ECOSYSTEM.label,
+    href: PATHS.ECOSYSTEM.path,
+    description: 'Learn about various ecosystem projects or submit your own',
+  },
+  {
+    label: PATHS.GOVERNANCE.label,
+    href: PATHS.GOVERNANCE.path,
+    description:
+      'Learn about Filecoin Improvement Proposals and other governance mechanisms',
+  },
+]
+
+const learnMoreItem = {
+  label: 'Learn more about Filecoin',
+  href: 'https://docs.filecoin.io/',
+}
+
+type LinkProps = {
+  label: string
+  description: string
+  href: string
+}
+
+function InternalLink({ label, description, href }: LinkProps) {
+  return (
+    <Link
+      href={href as Route}
+      className="inline-block w-full rounded-lg bg-brand-800 p-4 hover:bg-brand-700 focus:outline focus:outline-2 focus:outline-brand-100"
+      aria-label={`${label} page (internal link)`}
```

Comment:
Is an `aria-label` necessary when the link text is clear?

Reply:
I think so because the `Link` has two descendants, label and description, which get concatenated into one string in the accessibility tree. So the label makes it clearer hopefully.